docs/visualize/regionmap.asciidoc

```diff
@@ -0,0 +1,38 @@
+[[regionmap]]
+== Region Maps
+
+Region maps are thematic maps in which boundary vector shapes are colored using a gradient, with higher intensity colors indicating larger values and lower intensity colors indicating smaller values. 
+These are also known as choropleth maps. 
+
+image::images/regionmap.png[]
+
+
+=== Configuration
+
+To create such a map, you will need to configure a so-called "inner join", that will join the result of
+a terms-aggregation search-response from Elasticsearch to a reference vector file, based on a shared key.
+
+==== Data
+
+===== Metrics
+
+Select any of the supported _Metric_ or _Sibling Pipeline Aggregations_.
+
+===== Buckets 
+
+Configure a _Terms_ aggregation. The term is the "key" that will be used to join the results to the vector data on the map.
+
```

Comment:
The term is the _key_ that is used to join the results to the vector data on the map.